apps/fluent-tester/src/FluentTester/TestComponents/Menu/MenuTest.tsx

```diff
@@ -21,11 +21,13 @@ const MenuDefault: React.FunctionComponent = () => {
     <Stack style={stackStyle}>
       <Menu>
         <MenuTrigger>
-          <Button>Test</Button>
+          <Button disabled>Test</Button>
```

Comment:
Could we have the disabled trigger be a separate test? Rather than the "default" one on the test page?

Reply:
oh, yeah my bad, didn't mean to commit this.